src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+
+    public static void main(String[] args) throws IOException {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greeting();
+        chat();
     }
+
+    private static String readCommand() throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        String str = null;
+        System.out.println("Type something");
+        str = br.readLine().toLowerCase();
+        return str;
+    }
+
+    public static void greeting() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you");
+        System.out.println("____________________________________________________________");
+
+    }
+
+    public static void bye() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("____________________________________________________________");
+
+    }
+
+
+    public static void chat() throws IOException {
+        System.out.println("____________________________________________________________");
+        String str = readCommand();
+        System.out.println("____________________________________________________________");
+
+        if (str.equals("bye")) {
+            bye();
+        } else if (str.equals("add task")) {
+            System.out.println("____________________________________________________________");
+            System.out.println("Please add no more than 100 tasks");
+            System.out.println("____________________________________________________________");
+            addTaskScreen();
+        } else if (str.equals("list")) {
+            listTask();
+            chat();
+        }else {
+            System.out.println(str);
+            System.out.println("____________________________________________________________");
+            chat();
+        }
+    }
+
+
+    public static void addTaskScreen() throws IOException {
```

Comment:
There seems to be repeated calling of addTaskScreen at the end of each if-else statements. Perhaps, you might want to simply call it only after the entire if-else?

Another suggestion is to use switch statements for better readability.